_search-plugins/search-relevance/index.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Search relevance
+nav_order: 55
+has_children: false
+has_toc: false
+redirect_from:
+  - /search-plugins/search-relevance/
+---
+
+# Compare search results
+
+Compare search results is an experimental feature. For updates on the progress of Compare search results and other search relevance features, or if you want to leave feedback that could help improve the feature, join the [discussion on the OpenSearch forum](https://forum.opensearch.org/t/feedback-experimental-feature-compare-search-results/11331).    
+{: .warning}
+
+Compare search results is the first search relevance feature in OpenSearch. It lets you compare search results from two queries side by side to determine if one query produces better results than the other. Using this tool, you can evaluate search quality through experimenting with queries. 
+
+For example, you can see how results change when you apply one of the following query changes:
+
+- Weighting different fields differently
+- Different stemming or lemmatization strategies
+- Shingling
+
```

Comment:
Should these be ended with periods?

Reply:
Removed periods as per @vagimeli's[ comment ](https://github.com/opensearch-project/documentation-website/pull/1801#discussion_r1015805826)